Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift

```diff
@@ -865,3 +865,23 @@ final class SchemeDescriptorsGenerator: SchemeDescriptorsGenerating {
         }
     }
 }
+
+extension TestAction {
+    fileprivate static func empty(defaultBuildConfiguraiton: String) -> Self {
+        .init(
+            targets: [],
+            arguments: nil,
+            configurationName: "",
```

Comment:
Given nothing is going to be tested, it shouldn't be relevant I think 🤔 
If it's not the case, I think we don't have such information, because here we don't know anymore to which project each scheme belongs to